globals.d.ts

```diff
@@ -75,30 +118,30 @@ export interface BlockRow {
   sends?: number;
 }
 export interface StampRow {
-  stamp: number | null;
+  tx_hash: string;
   block_index: number;
-  cpid: string;
```

Comment:
this appears to have removed some important types for StampRow from the deb query such as

  divisible: boolean;
  keyburn: number | null;
  locked: number;
  stamp_base64: string;
  stamp_mimetype: string;
  stamp_url: string;
  supply: number;
  block_time: Date;
  tx_hash: string;
  ident: SUBPROTOCOLS;
  creator_name: string | null;
  stamp_hash: string;
  file_hash: string;
  floorPrice?: number | "priceless";
  marketCap?: number | "priceless";
  balance?: number | string;
  floorPriceUSD?: number | null;
  marketCapUSD?: number | null;
  recentSalePrice?: number | "priceless";
  unbound_quantity: number;
  
  which in some cases may be optional or specific to certain usages

Reply:
it looks like this is extended with StampWithSaleData etc. so it may not be an issue